### PR TITLE
[HOSTING-608] Create plugin-content-replace.yml

### PR DESCRIPTION
**Pull request by moxun1639**
# Description

Grant @moxun1639 permission to deploy content-replace-plugin

https://github.com/jenkinsci/content-replace-plugin

https://issues.jenkins-ci.org/browse/HOSTING-608

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

File: permissions/plugin-content-replace.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "content-replace"
+github: "jenkinsci/content-replace-plugin"
+paths:
+- "org/jenkins-ci/plugins/content-replace"
+developers:
+- "moxun1639"
```
Comment (daniel-beck) [inline permissions/plugin-content-replace.yml]: Wrong user name, please see readme.

File: permissions/plugin-content-replace.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: "content-replace"
+github: "jenkinsci/content-replace-plugin"
+paths:
+- "org/jenkins-ci/plugins/content-replace"
```
Comment (daniel-beck) [inline permissions/plugin-content-replace.yml]: Does not match POM content: https://github.com/jenkinsci/content-replace-plugin/blob/master/pom.xml#L11

Please see this repo's readme for how this works.

Comment (moxun1639) [reply permissions/plugin-content-replace.yml]: The “groupId” in pom.xml was removed, right?

Comment (daniel-beck) [reply permissions/plugin-content-replace.yml]: That should work.

**Pull request merged**